docs/CODEOWNERS

```diff
@@ -59,10 +59,11 @@
 /frame/contracts/ @athei
 
 # NPoS and election
-/frame/staking/ @kianenigma
-/frame/elections/ @kianenigma
-/frame/elections-phragmen/ @kianenigma
-/primitives/npos-elections/ @kianenigma
+/frame/election-provider-support/ @paritytech/staking-core
```

Comment:
I get a warning, but the team is there.. 🤔

Reply:
Fixed it for you, the team didn't had "write access to substrate"